src/SimplePDPService.sol

```diff
@@ -177,6 +179,9 @@ contract SimplePDPService is PDPListener, PDPRecordKeeper, Initializable, UUPSUp
         if (challengeCount < getChallengesPerProof()) {
             revert("Invalid challenge count < 5");
         }
+        if (provingDeadlines[proofSetId] == NO_PROVING_DEADLINE) {
+            revert("Proving not yet started");
```

Comment:
@ZenGround0 After thinking through it, I also think it's a good idea to revert this function if a proof does not fall in the challenge window i.e. in [0,`challengeWindow()`] epochs before the proving deadline.

Reply:
I think I agree.  It is also redundant and I worry it will confuse people down the line but I can add a longer comment above the check.

Reply:
Done